prov/efa/src/efa_mr.c

```diff
@@ -232,78 +243,136 @@ struct fi_ops efa_mr_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+/*
+ * Set core_access to FI_SEND | FI_RECV if not already set,
+ * set the fi_ibv_access modes and do real registration (ibv_mr_reg)
+ * Insert the key returned by ibv_mr_reg into efa mr_map and shm mr_map
+ */
+static int efa_mr_realloc_other(const void *buf, size_t len, uint64_t access,
+		      uint64_t offset, uint64_t flags, struct efa_mr *efa_mr,
+		      void *context)
+{
+	struct fi_mr_attr core_attr;
+	struct fi_mr_attr shm_attr;
```

Comment:
I think one attr is enough, this attr will be inserted to mr_map and used by fi_mr_reg(shm_domain).

We also do not need core_access, and we just added a comment here about efa lower provider only support IBV_ACCESS_LOCAL_WRITE